src/ssl.c

```diff
@@ -29977,6 +30023,36 @@ WOLFSSL_SESSION* wolfSSL_d2i_SSL_SESSION(WOLFSSL_SESSION** sess,
     return s;
 }
 
+/* Returns 1 if there is a session ticket associated with this WOLFSSL_SESSION.
+ *
+ * sess - pointer to WOLFSSL_SESSION struct
+ *
+ * Returns 1 if has session ticket, otherwise 0 */
```

Comment:
Can combine this comment and the one on line 30026.

Reply:
Addressed, thanks!